src/components/GoogleLogin.tsx

```diff
@@ -68,7 +58,6 @@ const GoogleLogin: FunctionComponent = () => {
         </IconButton>
       ) }
       onSuccess={ handleGoogleRespSuccess }
-      onFailure={ handleGoogleRespFailure }
```

Comment:
Removing this in favor of handling errors on our end through `onQueryStarted` in session api. Google will try to login on initial load and if it doesn't it'll throw this error this way